### ISSUE TITLE
Unexpected error occured

### ISSUE THREAD

**Issue by ethereumdegen** [OpenZeppelin/openzeppelin-upgrades#1085]
```
An unexpected error occurred:

Error: Failed to compile modified contracts for namespaced storage:

ParserError: Function, variable, struct or modifier declaration expected.
   --> contracts/LenderCommitmentForwarder/extensions/LenderCommitmentGroup/LenderCommitmentGroup_Smart.sol:141:6:
    |
141 |      ;
    |      ^



Please report this at https://zpl.in/upgrades/report. If possible, include the source code for the contracts mentioned in the errors above.
```

I am upgrading a contract that is a beacon proxy.  The original implementation did NOT have a library, and the new implementation does have a library.  The library is a separate read-only contract and is just referenced using an Import at the top of the implementation.   I believe trying to perform this kind of upgrade is throwing this error for me.  

Anyways this is my commit

https://github.com/teller-protocol/teller-protocol-v2/commit/c6b3f33aa57e6a54d8bdd7e3664cca52d0a66a7c

and my contracts that are most pertinent to this error are LenderCommitmentGroup_Smart.sol and UniswapPricingLibrary.sol

**Comment by ericglau** [comment]
Hi, can you check which version of `@openzeppelin/upgrades-core` is installed?  I don't see this error when using `@openzeppelin/upgrades-core@1.38.0`

**Comment by ethereumdegen** [comment]
Actually i solved the issue - i was trying to upgrade it as a regular proxy
but it is actually a beacon proxy. And im doing the upgrade via a defender
proposal so .. there be dragons .   Maybe we just brush this under a rug
for now lol

**Comment by ericglau** [comment]
This issue looks similar to https://github.com/OpenZeppelin/openzeppelin-upgrades/issues/1016 which was fixed in `@openzeppelin/upgrades-core@1.33.1`

I notice in your latest commit https://github.com/teller-protocol/teller-protocol-v2/pull/178/commits/2e7a3be4aa8390448b35e6498f1add73a2decb26 , you've updated the dependencies from `@openzeppelin/upgrades-core@1.30.0` to `@openzeppelin/upgrades-core@1.38.0`, which is the latest release and contains the above fix.

So it looks this was resolved by the dependency update.  Let me know if you have further issues though, thanks.
